### ISSUE TITLE
[docs] Power systems

### ISSUE THREAD

**Issue by jd-foster** [jump-dev/JuMP.jl#3267]
1) The timing comparison between adjustable costs has `elapsed time: 0.1309... seconds` while the _in-place_ version (where the claim is "faster than re-building the entire model") has `elapsed time: 0.2225... seconds`?

2) In the unit commitment formulation we have both the generator simple bounds and the UC bounds:
```julia
function solve_unit_commitment(generators::Vector, wind, scenario)
...
    @variable(model, generators[i].min <= g[i = 1:N] <= generators[i].max)
...
    # !!! New: add binary on-off variables for each generator
    @variable(model, u[i = 1:N], Bin)
    @constraint(model, [i = 1:N], g[i] <= generators[i].max * u[i])
    @constraint(model, [i = 1:N], g[i] >= generators[i].min * u[i])
...
```
which would imply that, since `generators[2].min = 300` then `u[2] == 1` regardless?

**Comment by odow** [comment]
1. Oh dear. This might just be variance in CI. If you run locally, it should be faster?
 2. Bug. Variables should not have bounds

**Comment by odow** [comment]
Huh. It seems that 1 isn't a fluke. It actually goes way back. Let me try again locally. This might be compilation latency

**Comment by jd-foster** [comment]
Yes, that was my thought too.

**Comment by jd-foster** [comment]
For 2)
## Before:
<img width="665" alt="image" src="https://user-images.githubusercontent.com/38274066/223000542-d9b3b2fc-9667-4b23-bc0a-cfca5ac9f0d5.png">

## After:
<img width="650" alt="image" src="https://user-images.githubusercontent.com/38274066/223000705-878ef351-6c7e-4399-b0ef-ab026ae22d1a.png">

**Comment by odow** [comment]
That new graph looks better/ You can see G2 switch on at the right time, and we're not spilling wind anymore.

**Comment by jd-foster** [comment]
Yes, that exact observation led me to look into the formulation.
